app/views/ideas/show.html.erb

```diff
@@ -81,3 +81,50 @@
 <% end %>
 <div class="govuk-!-padding-bottom-5"></div>
 <%= link_to 'Back', ideas_path %>
+
+<script>
```

Comment:
It would be nice to extract this out into a JS file, however I'd view this as minor issue for now.